ui.content.sample/src/main/content/META-INF/vault/filter.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <workspaceFilter version="1.0">
 
-    <!-- Content Filters -->
-    <filter root="/content/wknd" />
+    <!-- Content Filters
+    <filter root="/content/wknd" /> -->
```

Comment:
I think we need to leave this filter in. Most people want to deploy the full sample content. Yes it will overwrite existing content but users can always modify the filter as stated below...